ext/url/01_urlpattern.js

```diff
@@ -64,19 +64,21 @@ class URLPattern {
 
     const components = ops.op_urlpattern_parse(input, baseURL);
 
-    const keys = ObjectKeys(components);
-    for (let i = 0; i < keys.length; ++i) {
-      const key = keys[i];
+    this.#keys = ObjectKeys(components);
```

Comment:
The keys are a fixed array I think  (`protocol`, `hostname`, etc)

I wonder if we even need to do any `ObjectKeys` and store in a field, or if we can just use a const global array.

Reply:
That was my initial thought. I pushed the change to avoid `ObjectKeys` in the constructor